src/routers/TabRouter.js

```diff
@@ -129,6 +129,10 @@ export default (
       }
       if (activeTabIndex !== state.index) {
         // console.log(`${order.join('-')}: Normal navigation`, {lastIndex: state.index, newIndex: activeTabIndex});
+        if ( action.params ) {
+          state = {...state, routes: [...state.routes]};
+          state.routes[activeTabIndex] = {...state.routes[activeTabIndex], params: action.params};
```

Comment:
I think we shouldn't change the route unless the params have actually changed. Also, I think we may want to merge the params rather than overwrite them